css/style.css

```diff
@@ -0,0 +1,25 @@
+
+  body{
+    background: blue;
+    text-align: center;
+    opacity: 2;
+}
+#title{
+    text-align: center;
+    color: white;
+    font-size: 25px;
+    font-family:sans-serif
+}
+#description{
+    font-size: 16px;
+    color: white;
+}
+#Survey-form{
+    display: block;
+}
+label{
+    color: white;
+}
+input[type="name"],input[type="email"],input[type="number"]{
+    padding: 24px;
+}
```

Comment:
- Please make sure that your style.css pass  linters check. I recommend you to check stylesheet linters error by using this command  npx stylelint style.css --fix befor push in to github

Reply:
- [ OPTIONAL] Better you give some documentation to student instead of giving command to use so that the student can explore everything by himself , next time don't forget to give documentation link and use also emojis for human friendly :wink: :smile: